src/tests/ftest/launch.py

```diff
@@ -196,22 +196,19 @@ def set_test_environment(args):
     print("Before Environment:")
     for name in ("OFI_INTERFACE", "OFI_DOMAIN", "CRT_PHY_ADDR_STR"):
         print(" {0:>20}: {1}".format(name, os.environ.get(name, "No env")))
+    fi_info_path = os.path.join(os.path.sep, base_dir, bin_dir, "fi_info")
```

Comment:
`bin_dir` already has `base_dir` prefixed to it:
https://github.com/daos-stack/daos/blob/c4cd5e0403d23935b87ff1372fa6e1e919a9e3ab/src/tests/ftest/launch.py#L136

Reply:
Yes. Will fix and restart. Thanks. Good catch.